### PR TITLE
[Priest] Supporting new Dragonflight talents

### PR DESCRIPTION
**Pull request by DoxAshe**
Added Support for:
- Dark Void
- Mind Sear
- Mind Flay: Insanity
- Unfurling Darkness
- Inescapable Torment
- Shadow Word: Death 

While there are still a few talents that need statistics boxes, shadow priest is otherwise ready for Dragonflight with these changes.

### CODE REVIEW

File: src/common/TALENTS/priest.ts [hunk 1]
```diff
@@ -360,6 +360,12 @@ const talents = createTalentList({
     icon: 'spell_shadow_chilltouch',
     maxRanks: 2,
   },
+  INESCAPABLE_TORMENT_TALENT_DAMAGE: {
```
Comment (ToppleTheNun) [inline src/common/TALENTS/priest.ts]: Any manual additions for talents need to go in the SPELLS file for the class, as the TALENTS file will be overwritten any time the talents are regenerated.

**Review by ToppleTheNun — CHANGES_REQUESTED** [review]
Any manual additions for talents need to go in the SPELLS file for the class, as the TALENTS file will be overwritten any time the talents are regenerated. I don't see anything else that gives me pause, though.

File: src/common/TALENTS/priest.ts (hunk 1 above)
Comment (DoxAshe) [reply src/common/TALENTS/priest.ts]: That is great to know.  I moved the manual additions from TALENTS into SPELLS.

File: src/common/TALENTS/priest.ts [hunk 2]
```diff
@@ -448,13 +453,13 @@ const talents = createTalentList({
   VOID_ERUPTION_TALENT: {
     id: 228260,
     name: 'Void Eruption',
-    icon: 'spell_priest_void_blast',
+    icon: 'spell_priest_void-blast',
```
Comment (ToppleTheNun) [inline src/common/TALENTS/priest.ts]: This should still be `spell_priest_void_blast`.
```suggestion
    icon: 'spell_priest_void_blast',
```

File: src/common/TALENTS/priest.ts [hunk 3]
```diff
@@ -448,13 +453,13 @@ const talents = createTalentList({
...
     maxRanks: 1,
   },
   UNFURLING_DARKNESS_TALENT: {
     id: 341273,
     name: 'Unfurling Darkness',
-    icon: 'spell_priest_shadow_mend',
+    icon: 'spell_priest_shadow-mend',
```
Comment (ToppleTheNun) [inline src/common/TALENTS/priest.ts]: This should still be `spell_priest_shadow_mend`.
```suggestion
    icon: 'spell_priest_shadow_mend',
```

**Review by ToppleTheNun — CHANGES_REQUESTED** [review]
Once the two comments on talents are done, this should be good to go.

File: src/common/TALENTS/priest.ts (hunk 2 above)
Comment (DoxAshe) [reply src/common/TALENTS/priest.ts]: I've undone all of the changes to TALENTS

File: src/common/TALENTS/priest.ts (hunk 3 above)
Comment (DoxAshe) [reply src/common/TALENTS/priest.ts]: I've undone all the changes to TALENTS

**Review by ToppleTheNun — APPROVED** [review]

**Pull request merged**